pytest_falcon/plugin.py

```diff
@@ -132,12 +132,15 @@ def fake_request(self, path, **kwargs):
         body = self.app(create_environ(path, **kwargs), resp)
         resp.headers = resp.headers_dict
         resp.status_code = int(resp.status.split(' ')[0])
-        resp.body = body[0] if body else b''
+        try:
```

Comment:
Seems to me that the try is unnecessary. Something like that should handle both scenario, no?

```
        resp.body = b''.join(list(body)) if body else b''
```